<launch>
  <include file="$(find pr2_arm_navigation_actions)/launch/head_monitor_right_arm.launch"/>	

  <node pkg="move_arm" type="move_arm_simple_action" output="screen" name="move_right_arm">    

    <remap from="robot_description" to="robot_description" />
    <remap from="joint_state" to="joint_states" />
    <remap from="arm_ik" to="pr2_right_arm_kinematics/get_constraint_aware_ik" />	
    <remap from="arm_fk" to="pr2_right_arm_kinematics/get_fk" />	
    <remap from="filter_trajectory" to="trajectory_filter/filter_trajectory_with_constraints" />
    <!--remap from="filter_trajectory" to="chomp_planner_longrange/filter_trajectory_with_constraints" /-->
    
    <remap from="get_trajectory_validity" to="environment_server/get_trajectory_validity" />	
    <remap from="get_environment_safety" to="environment_server/get_environment_safety" />	
    <remap from="get_execution_safety" to="environment_server/get_execution_safety" />	
    <remap from="get_group_info" to="environment_server/get_group_info" />	
    <remap from="get_robot_state" to="environment_server/get_robot_state" />	
    <remap from="get_state_validity" to="environment_server/get_state_validity" />	

    <remap from="head_monitor_action" to="head_monitor_right_arm/monitor_action" />
    <remap from="head_look_action" to="head_monitor_right_arm/look_action" />

    <param name="group" type="string" value="right_arm" />
    <param name="ik_allowed_time" type="double" value="2.0" />
    <param name="trajectory_filter_allowed_time" type="double" value="2.0" />
    <param name="group" type="string" value="right_arm" />
    <param name="controller_action_name" type="string" value="/r_arm_controller/joint_trajectory_action" />

    <param name="head_monitor_link" type="string" value="" />
    <param name="head_monitor_link_x" type="double" value="0.10" />
    <param name="head_monitor_time_offset" type="double" value="1.5" />
    <param name="head_monitor_max_frequency" type="double" value="3.0" />

    <param name="pause_allowed_time" type="double" value="5.0" />

  </node>
</launch>
